frontend/src/components/pages/RegistrantExperience/RegistrationResult/RegistrationErrorModal.tsx

```diff
@@ -0,0 +1,60 @@
+import {
+  Button,
+  HStack,
+  Modal,
+  ModalContent,
+  ModalFooter,
+  ModalHeader,
+  ModalOverlay,
+  Text,
+} from "@chakra-ui/react";
+import React from "react";
+import { ModalBody } from "react-bootstrap";
+
+type RegistrationErrorModalProps = {
+  title: string;
+  message: string;
+  onConfirm: () => void;
+  isOpen: boolean;
+  onClose: () => void;
+};
+
+const RegistrationErrorModal = ({
+  title,
+  message,
+  onConfirm,
+  isOpen,
+  onClose,
+}: RegistrationErrorModalProps): React.ReactElement => {
+  return (
+    <Modal isOpen={isOpen} onClose={onClose} isCentered>
+      <ModalOverlay />
+      <ModalContent>
+        <ModalHeader>
+          <Text>{title}</Text>
+        </ModalHeader>
+        <ModalBody>
+          <Text px="8px">{message}</Text>
```

Comment:
not sure why there was different padding by default on header vs. body, can someone else check what this looks like on your screen?